### PR TITLE
Add response schemas

### PR DESCRIPTION
**Pull request by pavonism**

### CODE REVIEW

File: adopterAPI.json [hunk 1]
```diff
@@ -354,40 +427,152 @@
...
+        },
+        "additionalProperties": false
+      },
+      "ApplicationStatusDTO": {
+        "enum": [
+          "Created",
+          "Accepted",
+          "Rejected",
+          "Withdrawn",
+          "Deleted"
```
Comment (Meeso1) [inline adopterAPI.json:536]: Do we need `"Deleted"` here? When would we use this? I don't think we can set it from API.

File: adopterAPI.json [hunk 2]
```diff
@@ -420,6 +605,11 @@
             "type": "string",
             "format": "uuid",
             "nullable": true
+          },
+          "adopterId": {
+            "type": "string",
+            "format": "uuid",
+            "nullable": true
           }
```
Comment (Meeso1) [inline adopterAPI.json]: We don't need this. We decided to just pull this from JWT token in some previous PR.

**Review by Meeso1 — COMMENTED** [review]
Generally LGTM

File: adopterAPI.json (hunk 1 above)
Comment (sivvak) [reply adopterAPI.json:536]: I think ``"Deleted"`` is identical to ``"Withdrawn"`` according to the documentation and can be safely deleted. Someone correct me if I'm wrong, please.

**Review by sivvak — COMMENTED** [review]
In general, looks good to me, I only suggest deleting an unnecessary value in `ApplicationStatusDTO` and adopterId in `PostApplicationRequest`.

File: adopterAPI.json (hunk 1 above)
Comment (pavonism) [reply adopterAPI.json:536]: It was supposed to be used by administrator to soft-delete application when it was reported

**Review by kubaurban — APPROVED** [review]

**Review by sivvak — APPROVED** [review]

**Pull request merged**